Pod/VOKNavigationHelper/VOKNavigationHelper.m

```diff
@@ -0,0 +1,122 @@
+//
+//  VOKNavigationHelper.m
+//  VOKUtilities
+//
+//  Created by Brock Boland on 1/24/17.
+//  Copyright (c) 2017 Vokal. All rights reserved.
+//
+
+#import "VOKNavigationHelper.h"
+
+@implementation VOKNavigationHelper
+
++ (void)clearExistingViewsAndAnimateToViewController:(UIViewController *)viewController
+                                            duration:(NSTimeInterval)duration
+                                             options:(UIViewAnimationOptions)options
+                                          completion:(void (^ __nullable)(BOOL finished))completion
+{
+    // Enforce the non-null
+    NSParameterAssert(viewController != nil);
```

Comment:
I don't think you need the `!= nil` part.  I've typically used it like `NSParameterAssert(viewController);`.